amftrack/pipeline/functions/image_processing/experiment_util.py

```diff
@@ -39,3 +67,94 @@ def distance_point_edge(point: coord, edge: Edge, t: int, step=1):
 
     pixel_list = edge.pixel_list(t)
     return distance_point_pixel_line(point, pixel_list, step)
+
+
+def aux_plot_edge(
+    edge: Edge, t: int, mode=0, points=10
+) -> Tuple[np.array, List[coord]]:
+    """
+    This intermediary function returns the data that will be plotted.
+    See plot_edge for more information.
+    :return: image, coordinate of all the points in image
+    """
+    # TODO(FK): use a mask instead of points
+    exp = edge.experiment
+    number_points = 10
+    # Fetch coordinates of edge points in general referential
+    if mode == 0:
+        list_of_coord = [edge.begin.pos(t), edge.end.pos(t)]
+    else:
+        list_of_coord = edge.pixel_list(t)
+        if mode == 2:
+            pace = len(list_of_coord) // number_points
+            l = [list_of_coord[i * pace] for i in range(number_points)]
+            l.append(list_of_coord[-1])
+            list_of_coord = l
+
+    # Image index: we take the first one we find for the first node
+    x, y = list_of_coord[0]
+    try:
+        i = exp.find_im_indexes_from_general(x, y, t)[0]
+    except:
+        raise Exception("There is no image for this Edge")
+    # Convert coordinates to image coordinates
+    list_of_coord_im = [exp.general_to_image(coord, t, i) for coord in list_of_coord]
+    # Fetch the image
+    im = exp.get_image(t, i)
+    return im, list_of_coord_im
+
+
+def plot_edge(edge: Edge, t: int, mode=2, points=10):
+    """
+    Plot the Edge in its source images, if one exists.
+    :WARNING: If the edge is accross two image, only a part of the edge will be plotted
```

Comment:
Maybe would be worth to workout some improvement of plot_raw_plus (and potentially rewrite it) to make it possible to plot accross images in full quality. 
Now that you have a bit of experience, maybe it would be worth sitting together to design an ideal "plotting" library

Reply:
Yes, I aggree, we can have a talk over it tomorrow

Reply:
I'll make a small list of all the plotting function that I think I might need